arcus_orientation.md

```diff
@@ -841,32 +849,32 @@ It is a violation of the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-us
 </div>
 
 Many researchers use GitHub to archive and share code for their analyses, often as part of the process of publishing a scientific paper.
-You can always use repositories on CHOP's Enterprise GitHub from your Arcus Lab, whether or not you have internet disabled, but CHOP's Enterprise GitHub is insufficient for sharing code as part of a publication because it's visible only to people with CHOP credentials. 
+You can use repositories on CHOP's Enterprise GitHub from your Arcus Lab in Internet Disabled mode, but CHOP's Enterprise GitHub is insufficient for sharing code as part of a publication because it's visible only to people with CHOP credentials. 
 
-If you plan to use a repository on [www.github.com](https://github.com/)(public GitHub) to share the R code from your Arcus Lab, you will need to enable internet when it's time to push commits to the repository. 
-Remember that you do not need an internet connection to do most actions using Git, such as creating new commits or merging branches, so you can wait until it's time to push to [enable internet](#lab-status-pane), and then disable internet again afterward. 
+If you plan to use a repository on [www.github.com](https://github.com/) (public GitHub) to share the R code from your Arcus Lab, you will need to enable internet when it's time to push commits to the repository. 
+Remember that you do not need an internet connection to do most actions using Git, such as creating new commits or merging branches, so you can wait until it's time to push to [enable internet](#switching-modes), and then return to Internet Disabled mode again afterward. 
```

Comment:
should we be referring to enabling internet as using "Internet Enabled mode" similar to how we refer to the opposite setting as using "Internet Disabled mode". If we do end up doing this, we should take another look through the full file to make sure we're being consistent with that. 


```suggestion
Remember that you do not need an internet connection to do most actions using Git, such as creating new commits or merging branches, so you can wait until it's time to push to turn on [Internet Enabled mode](#switching-modes), and then return to Internet Disabled mode again afterward. 
```